src/diagnostics/MOM_spatial_means.F90

```diff
@@ -47,6 +47,52 @@ function global_area_mean(var, G, scale)
 
 end function global_area_mean
 
+!> Return the global area mean of a variable. This uses reproducing sums.
+function global_area_mean_v(var, G)
+  type(ocean_grid_type),             intent(in)  :: G    !< The ocean's grid structure
+  real, dimension(SZI_(G), SZJB_(G)), intent(in)  :: var  !< The variable to average
+
+  real, dimension(SZI_(G), SZJ_(G))              :: tmpForSumming
+  real :: global_area_mean_v
+  integer :: i, j, is, ie, js, je, isB, ieB, jsB, jeB
+
+  is = G%isc ; ie = G%iec ; js = G%jsc ; je = G%jec
+  isB = G%iscB ; ieB = G%iecB ; jsB = G%jscB ; jeB = G%jecB
+
+  tmpForSumming(:,:) = 0.
+  do J=js,je ; do i=is,ie
+     tmpForSumming(i,j) = G%areaT(i,j)/max(1.e-20,&
```

Comment:
Fix formatting.

Reply:
In this expression, I think that we would prefer to have division at the end, or else put parentheses around the area divided by the masks, so that someone reading the code does not need to remember the order of precedence in Fortran between multiplication and division (they are the same), or whether the Fortran order of operations goes left to right or right to left.  The same comment would apply in global_area_mean_u().

Reply:
The most recent code push should address this.